collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -341,8 +331,9 @@ private void downloadFormList() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putBoolean(BUNDLE_TOGGLED_KEY, mToggled);
-        outState.putInt(BUNDLE_SELECTED_COUNT, selectedItemCount());
+        // TODO: Is it safe to remove BUNDLE_TOGGLED_KEY ?
```

Comment:
I don't see any problem with removing the writing out and reading back in but maybe I'm missing something. What do you think the issue could be? Removing the BUNDLE_TOGGLED_KEY constant is also fine as it's just an identifier for the key.

I was talking about removing the persisted key/value pair but actually I don't think `onSaveInstanceState` persists values beyond app termination.